windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
> So you can just double click the project file and it opens in Visual Studio 2013 just fine?

Only if VS2015 is not installed, else you have to explicitely open in VS2013, but in any case it works ok on the 2 machines I tried it on. No prompt for downgrading, no changes to the project file. It's news to me as well, in fact I'm sure tons of people out there have no idea, but it's good news.